### ISSUE TITLE
Typescript error: Not export *.d.ts types

### ISSUE THREAD

**Issue by victorfernandesraton** [nullstack/nullstack#302]
I currently using ```0.17``` version and i faced an error when try import types like ```NullstackClientContext``` and when using typescript to analyze the project, I found this:

```
An import path cannot end with a '.d.ts' extension. Consider importing './ClientContext' instead.ts(2691)
```

And when I figure out to resolve i see in ```package.json``` types point to /types/index.d.ts but not found them

i suggest creating an script with separated tsconfig to emit types only and generate only index.d.ts files instead of various .d.ts files for every item in the source

**Comment by GuiDevloper** [comment]
I received that one too!
The solution is in fact the one pointed there:
- Our types were manually written and the paths to them are wrongly including the extension `.d.ts`. (eg. `./ClientContext.d.ts`)
- Remove the extensions from the imports and everything works fine! (eg. `./ClientContext`)

PR are welcome! :)

**Comment by Mortaro** [comment]
Thanks for the report 💜
This has been fixed on 0.17.3 please reopen if encounter the error again
